homeassistant/components/reolink/host.py

```diff
@@ -273,6 +274,18 @@ async def register_webhook(self) -> None:
         webhook_path = webhook.async_generate_path(event_id)
         self._webhook_url = f"{base_url}{webhook_path}"
 
+        if base_url.startswith("https"):
+            ir.async_create_issue(
+                self._hass,
+                DOMAIN,
+                "https_webhook",
+                breaks_in_ha_version="2023.2.0",
```

Comment:
I don't think there should be a `breaks_in_ha_version` item. This is a problem already and for the future, right?

Reply:
yes you are right, I have removed the breaks_in_ha_version